main 2.py

```diff
@@ -0,0 +1,130 @@
+def mainScreen():
+    print("1. Save a New Entry")
+    print("2. Search By Id")
+    print("3. Print Ages Average")
+    print("4. Print All Names")
+    print("5. Print All IDs")
+    print("6. Print All Entries")
+    print("7. Print Entry By index")
+    print("8. Exit")
+
+
+def checkIfNumber(user_input):
+    char_flag = True
+    for num in user_input:
+        if num != "9" and num != "8" and num != "7" and num != "6" and num != "5" and num != "4" and num != "3" and num != "2" and num != "1" and num != "0":
+           char_flag = False
+    return char_flag
+    
+    
+def userChoseone(my_dic):
+        while True:
+            user_id = input("Enter Id Number ")
+            user_age = input("Enter User Age ")
+            if checkIfNumber(user_id) and checkIfNumber(user_age) == True:
```

Comment:
ההשוואה כאן ל True היא רק עבור checkIfNumber(user_age) == True. תשים לב שכשכותבים and, המתרגם מצפה לשני ביטויים בוליאניים, אחת בכל צד.
למשל אם היינו רוצים לבדוק שX ו-Y שווים ל1, אי אפשר לעשות
if x and y == 1
אלא
if x == 1 and y == 1